workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/VisualHolder.kt

```diff
@@ -0,0 +1,24 @@
+package com.squareup.workflow1.ui
+
+@WorkflowUiExperimentalApi
+public interface VisualHolder<in RenderingT, out VisualT> {
```

Comment:
It seems like some of the kdoc for `VisualFactory.createOrNull` should be here as well.

Reply:
Uh oh. I think this and `VisualHolder` leaked in from an unrelated PR.